web/src/js/components/Modes/Regular.tsx

```diff
@@ -33,7 +39,11 @@ export default function Regular() {
                     onEditDone={(port) => handlePortChange(port)}
                 />
             </ModeToggle>
-            {error && <div className="mode-error text-danger">{error}</div>}
+            {(error || error_state) && (
```

Comment:
Let's rename `error` to `ui_error` and `error_state` to `backend_error` to make more clear what's going on here?

Reply:
make sense sure :)